### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by aakahlow**
Here is the rtd version of docs for this PR: https://gem5art.readthedocs.io/en/documentation-updates

### CODE REVIEW

File: docs/artifacts.md [hunk 1]
```diff
@@ -136,6 +132,28 @@ import gem5art.artifact
 for i in gem5art.artifact.getByName("gem5"):print(i)
 ```
 
+## Downloading from the Database
```
Comment (powerjg) [inline docs/artifacts.md:135]: Could you add a suggestion to use they python interactive terminal? Something like:

```
A good way to search and download items from the database is by using the Python interactive shell.
You can search the database with the functions provided by the `artifact` module (e.g., [`getByName`](LINK), [`getByType`](LINK), etc.).
Then, once you've found the ID of the artifact you'd like to download, you can call [`downloadFile`](LINK).
See the example below.

(Note: put triple backticks here...)
$ python
Python 3.6.8 (default, Oct  7 2019, 12:59:55) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from gem5art.artifact import *
>>> db = ArtifactDB()
>>> for i in getDiskImages(limit=2): print(i)
... 
ubuntu
    id: d4a54de8-3a1f-4d4d-9175-53c15e647afd
    type: disk image
    path: disk-image/ubuntu-image/ubuntu
    inputs: packer:fe8ba737-ffd4-44fa-88b7-9cd072f82979, fs-x86-test:94092971-4277-4d38-9e4a-495a7119a5e5, m5:69dad8b1-48d0-43dd-a538-f3196a894804
    Ubuntu with m5 binary installed and root auto login
ubuntu
    id: c54b8805-48d6-425d-ac81-9b1badba206e
    type: disk image
    path: disk-image/ubuntu-image/ubuntu
    inputs: packer:fe8ba737-ffd4-44fa-88b7-9cd072f82979, fs-x86-test:5bfaab52-7d04-49f2-8fea-c5af8a7f34a8, m5:69dad8b1-48d0-43dd-a538-f3196a894804
    Ubuntu with m5 binary installed and root auto login
>>> for i in getLinuxBinaries(limit=2): print(i)
... 
vmlinux-5.2.3
    id: 8cfd9fbe-24d0-40b5-897e-beca3df80dd2
    type: kernel
    path: linux-stable/vmlinux-5.2.3
    inputs: fs-x86-test:94092971-4277-4d38-9e4a-495a7119a5e5, linux-stable:25feca9a-3642-458e-a179-f3705266b2fe
    Kernel binary for 5.2.3 with simple config file
vmlinux-5.2.3
    id: 9721d8c9-dc41-49ba-ab5c-3ed169e24166
    type: kernel
    path: linux-stable/vmlinux-5.2.3
    inputs: npb:85e6dd97-c946-4596-9b52-0bb145810d68, linux-stable:25feca9a-3642-458e-a179-f3705266b2fe
    Kernel binary for 5.2.3 with simple config file
>>> from uuid import UUID
>>> db.downloadFile(UUID('8cfd9fbe-24d0-40b5-897e-beca3df80dd2'), 'linux-stable/vmlinux-5.2.3')


```

**Review by powerjg — APPROVED** [review]
Looks good. I have one small addition, but it can be another PR, if that's easier.

File: docs/artifacts.md (hunk 1 above)
Comment (aakahlow) [reply docs/artifacts.md:135]: Updated!

Comment (powerjg) [reply docs/artifacts.md:135]: Looks great to me. Ready to push :)

**Pull request merged**